### DISCUSSION TITLE
⛔ 🚀 Feature Request: Forms - Select Placeholder & Radio popout bug

### DISCUSSION THREAD

**Discussion by Dentalicious** [elementor/elementor#21420] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

1) There is no option for placing a placeholder for the select field in forms.

2) If there is a radio button in the form field in a popout, clicking anywhere outside the radio button closes the popout

3) The number field does not enforce a restriction placed on it. I placed a restriction of minimum and maximum 4 numbers for the field, however, the test form still got sent when I inserted 6 digits in the form field.

### Describe the solution you'd like

1) It would be nice if there was an option for a placeholder for the select field in forms.

2) Ideally, the popout shouldn't close if one selects anywhere outside the radio buttons within the popout except the X button.

3) The digit restriction in the numbers form field must be enforced.

**Comment by nicholaszein** [comment]
Hello there @Dentalicious!

thanks for submitting your feature request! 🙏 

### 🗃️ Please understand that while we do welcome new feature requests, we can't track multiple requests per thread. 

Please follow our guidelines for submitting a new [🚀 Feature Request](https://github.com/elementor/elementor/discussions/new?category=feature-request), and submit only ☝️ **_<ins>one request per thread</ins>_** so we can track them better and allow users to vote only for the features they want to see implemented.

Kind regards
